### DISCUSSION TITLE
Content Editor & Generic Editor Plugin service Failure

### DISCUSSION THREAD

**Discussion by prabhunaveen** [Sunbird-Knowlg/Community#68] [Bugs]
@vinukumar-vs @sharathkashyap 
As part of release-5.0.0, The build for Content editor & Generic Editor Plugin service is Failed with below error.Kindly provide fix for this issue.

`npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.0.0 (node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.3.2 (node_modules/postcss-cli/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@~2.3.2 (node_modules/watchpack/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@2.3.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@^1.2.7 (node_modules/watchpack-chokidar2/node_modules/chokidar/node_modules/fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.2.13: wanted {"os":"darwin","arch":"any"} (current: {"os":"linux","arch":"x64"})
npm WARN postcss-cli@8.2.0 requires a peer of postcss@^8.0.0 but none was installed.
npm WARN postcss-load-config@3.1.4 requires a peer of postcss@>=8.0.9 but none was installed.
npm WARN postcss-load-config@3.1.4 requires a peer of ts-node@>=9.0.0 but none was installed.
npm WARN postcss-reporter@7.0.5 requires a peer of postcss@^8.1.0 but none was installed.
npm ERR! Linux 4.15.0-1113-azure
npm ERR! argv "/usr/local/lib/node-v6.17.1-linux-x64/bin/node" "/usr/bin/npm" "install"
npm ERR! node v6.17.1
npm ERR! npm  v3.10.10
npm ERR! path /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/.staging/@types/minimatch-8b3a555e/package.json
npm ERR! code ENOTDIR
npm ERR! errno -20
npm ERR! syscall open

npm ERR! ENOTDIR: not a directory, open '/var/lib/jenkins/workspace/Build/Plugins/GenericEditor/node_modules/.staging/@types/minimatch-8b3a555e/package.json'
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

npm ERR! Please include the following file with any support request:
npm ERR!     /var/lib/jenkins/workspace/Build/Plugins/GenericEditor/npm-debug.log`

@AmiableAnil @anandvarada @srajasimman

**Comment by vaibhavbhuva** [comment]
@prabhunaveen Answered here - https://github.com/orgs/Sunbird-Ed/discussions/97#discussioncomment-3555656
